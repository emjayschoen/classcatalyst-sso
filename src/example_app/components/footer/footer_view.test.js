import React from 'react';
import { shallow } from 'enzyme';
import FooterView from './footer_view';

describe('<FooterView />', () => {
  let subject;
  beforeEach(() => {
    subject = shallow(<FooterView />);
  });

  it('renders correctly', () => {
    expect(subject).toMatchSnapshot();
  });
});
