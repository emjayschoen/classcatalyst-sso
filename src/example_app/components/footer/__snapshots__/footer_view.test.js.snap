// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FooterView /> renders correctly 1`] = `
<styled.div>
  <span>
    © Monkey and 
    <span
      aria-label="Banana"
      role="img"
    >
      🍌
    </span>
    , Inc.
  </span>
</styled.div>
`;
